static/app/views/performance/table.tsx

```diff
@@ -160,6 +198,28 @@ class Table extends React.Component<Props, State> {
       );
     }
 
+    if (field === 'id' || field === 'trace') {
+      const generateLink = field === 'id' ? generateTransactionLink : generateTraceLink;
+      const target = generateLink(eventView.name as string)(
```

Comment:
You probably should not be using `eventView.name` here to get the transaction name as that can contain an arbitrary string and is likely to cause problems in the future if it ever changes.

Reply:
Updated to use `transactionName` passed from parent components. `transactionName` is created by calling `getTransactionName` on `location`. This seemed to be a common way of deducing `transactionName`.